modules/csp-overview.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plugin/content-security-policy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="content-security-policy-overview_{context}"]
+= Content Security Policy (CSP) overview
+
+A Content Security Policy (CSP) is delivered to the browser in the `content-security-policy` response header. The policy is specified as a series of directives aqnd values. Each directive type serves a different purpose, and each directive can hace a list of values representing alllowed sources.
+
+[IMPORTANT]
+====
+The console currently uses `Content-Security-Policy-Report-Only`, so the browser will only warn about CSP violations in the web console and enforcment of CSP policies will be limited.
+====
+
+[id="content-security-policy-overview_{context}"]
+== Key features of `ConsolePluginCSP`
+
+[discrete]
+=== Directive Types
+
+The supported directive types include `DefaultSrc`, `ScriptSrc`, `StyleSrc`, `ImgSrc`, and `FontSrc`, which allows plugins to specify valid sources for loading different types of content. Each directive type serves a different purpose, `ScriptSrc` defines valid JavaScript sources, while `ImgSrc` controls where images can be loaded from.
```

Comment:
we are also backporting the `ConnectSrc` directive, but that under way - https://github.com/openshift/console/pull/14701

Reply:
Okay cool! I'll add, but comment out so I remember to add it in when it is available.